document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreQueryV1Test.java

```diff
@@ -626,6 +626,34 @@ public void testQueryWithFunctionalLeftHandSideInFilter(final String dataStoreNa
     testCountApi(dataStoreName, query, "query/test_aggr_functional_lhs_in_filter_response.json");
   }
 
+  @ParameterizedTest
+  @MethodSource("databaseContextBoth")
+  public void testFindWithFunctionalLeftHandSideFilter(final String dataStoreName)
+      throws IOException {
+    final Datastore datastore = datastoreMap.get(dataStoreName);
+    final Collection collection = datastore.getCollection(COLLECTION_NAME);
+
+    final Query query =
+        Query.builder()
+            .addSelection(IdentifierExpression.of("item"))
+            .setFilter(
+                RelationalExpression.of(
+                    FunctionExpression.builder()
+                        .operator(MULTIPLY)
+                        .operand(IdentifierExpression.of("quantity"))
+                        .operand(IdentifierExpression.of("price"))
+                        .build(),
+                    GT,
+                    ConstantExpression.of(50)))
+            .addSort(IdentifierExpression.of("item"), DESC)
+            .build();
+
+    final Iterator<Document> resultDocs = collection.find(query);
+    assertDocsAndSizeEqual(
+        dataStoreName, resultDocs, "query/test_aggr_functional_lhs_in_filter_response.json", 3);
```

Comment:
nit (upto you): shall we remove `aggr` from file name `test_aggr_functional_lhs_in_filter_response.json` -> `test_functional_lhs_in_filter_response.json` as it is used now both with `find` API and `aggregate` API ?